app/components/utility/cookie_banners_component.rb

```diff
@@ -35,9 +37,15 @@ def valid_namespace?
     current_namespace.eql?(CANDIDATE_INTERFACE) || current_namespace.eql?(PROVIDER_INTERFACE)
   end
 
+  def cookies_have_not_been_set?
```

Comment:
last nit! maybe clearer to call this method `cookies_have_already_been_set?` and then we can avoid the slight double negative feeling that comes with returning the outcome of `none?`

Reply:
would this be a bit confusing, if they have already been set why would the component need to be rendered?